pilot/pkg/networking/core/v1alpha3/route/route.go

```diff
@@ -315,10 +317,7 @@ func translateRoute(in *networking.HTTPRoute,
 		}
 
 		if in.Mirror != nil {
-			n, err := nameF(in.Mirror)
-			if err != nil {
-				return nil, err
-			}
+			n, _ := nameF(in.Mirror)
```

Comment:
We probably should ignore this error and should at least log it (since we don't want to return here anymore).